paasta_tools/cli/cmds/validate.py

```diff
@@ -642,5 +676,7 @@ def paasta_validate(args):
     """
     service_path = get_service_path(args.service, args.yelpsoa_config_root)
     service = args.service or guess_service_name()
-    if not paasta_validate_soa_configs(service, service_path):
+    verbose = args.verbose
+
+    if not paasta_validate_soa_configs(service, service_path, verbose):
```

Comment:
nit: i'd personally inline this - but if you think it's better to be consistent feel free to drop this comment